fpdf/html.py

```diff
@@ -301,9 +303,21 @@ def _insert_td(self, data=""):
         bgcolor = color_as_decimal(self.td.get("bgcolor", self.tr.get("bgcolor", "")))
         # parsing table header/footer (drawn later):
         if self.thead is not None:
-            self.theader.append(((width, height, data, border, 0, align), bgcolor))
+            # The ln=None (after new[xy]) needs cleanup after ln is removed
+            self.theader.append(
+                (
+                    (width, height, data, border, XPos.RIGHT, YPos.TOP, None, align),
```

Comment:
Do you think you could use this opportunity to the first element of the 2-tuples in `self.theader` list to be `dict`? This way, in `output_table_footer()`, `self.pdf.cell(**cell)` could be called, and the code would not rely on the ordering of arguments passed to this method, which is a bit brittle.

Reply:
I tried to touch this part as little as possible for the moment... But that is indeed a very good idea in this case (especially when using 'DEPRECATED' instead of `None`).